src/vs/code/node/sharedProcess/contrib/voiceTranscriber.ts

```diff
@@ -136,6 +143,12 @@ class SlidingWindowVoiceTranscriber extends VoiceTranscriber {
 
 		this.port.postMessage(this.transcribedResults.join(' '));
 	}
+
+	override dispose(): void {
+		super.dispose();
+
+		this.data = new Float32Array(0);
+	}
 }
```

Comment:
nit: overriding dispose is a little dangerous as it's not enforced that `super.dispose()` is called. Can you do this instead in the ctor?

```ts
this._register(toDisposable(() => this.data = new Float32Array(0)));
```